packages/select/src/components/select/select2.md

```diff
@@ -286,6 +288,7 @@ const renderMenu: ItemListRenderer<Film> = ({ items, itemsParentRef, query, rend
             <MenuItem
                 disabled={true}
                 text={`Found ${renderedItems.length} items matching "${query}"`}
+				roleConfig="listoption"
```

Comment:
bad indentation; these should be spaces instead of tabs

Reply:
fixed i think